CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ Possible log types:
 
 - [changed] Minimal required Rust version 1.20.0 (#68)
 - [changed] Update `log` crate to version 0.4
+- [changed] Introduce `IssueReportChannel` enum (#4)
```

Comment:
Maybe rephrase it to "Use an enum (`IssueReportChannel`) for the issue_report_channels field" so it's clear which SpaceAPI field is affected?